### PR TITLE
Shadcn migration - more components

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Migrate the following components to TW.

- Banners/UpgradeBannerNotification.tsx
- Banners/BugBountyBanner/index.tsx
- CopyToClipboard.tsx
- EnvWarningBanner.tsx
- EthVideo.tsx
- FeaturedText/index.tsx
- IdAnchor.tsx
- LearningToolsCardGrid.tsx
- MarkdownImage.tsx
- MergeArticleList.tsx
- StatErrorMessage.tsx
- TranslationChartImage/index.tsx
- YouTube.tsx

### CODE REVIEW

File: src/components/IdAnchor.tsx
Comment (wackerow) [inline src/components/IdAnchor.tsx:1]: image

Looks like most of this was missed... Will post a sub-branch PR with a fix real quick

**Review by wackerow — CHANGES_REQUESTED** [review]
Overall looking good, just spotted the IdAnchor needed updating

File: src/components/IdAnchor.tsx
Comment (wackerow) [reply src/components/IdAnchor.tsx:1]: #13980

Comment (pettinarip) [reply src/components/IdAnchor.tsx:1]: Ahh nice! good catch @wackerow thanks for the PR!

File: src/components/MarkdownImage.tsx [hunk 1]
```diff
@@ -42,20 +41,25 @@ const MarkdownImage = ({
...
-    <Flex as="span" justify="center">
-      <Link href={transformedSrc} target="_blank" rel="noopener" locale={false}>
-        <Image
+    <span className="flex justify-center">
+      <InlineLink
+        href={transformedSrc}
+        target="_blank"
+        rel="noopener"
+        locale={false}
+      >
```
Comment (wackerow) [inline src/components/MarkdownImage.tsx:50]: I'm not sure we need the `target` and `rel` any more.... we're using `ui/Link` now which should parse this with `const isExternal = url.isExternal(href)`

```suggestion
      <InlineLink href={transformedSrc} locale={false}>
```

Comment (wackerow) [reply src/components/MarkdownImage.tsx:50]: Never mind! I see we're feeding local images to this so this override is needed. Approving =)

**Review by wackerow — APPROVED** [review]
Looks good! Thanks @pettinarip!

**Pull request merged**